### PR TITLE
Fix error while overwriting version details while with cve details

### LINKED ISSUE #10: Error while overwriting version details while with cve details.
## Description

The `cve_details` field in the version details file is not overwritten properly if it is carrying data.

## Expected Result

image

## Actual Result

image

### PR DESCRIPTION
**Pull request by sudhirverma**
Fix: #10

### CODE REVIEW

File: besecure_developer_toolkit/src/CreateVersionData.py [hunk 1]
```diff
@@ -44,11 +44,13 @@ def cleanup(self):
     def overwrite_version_data(self, f, version_data_new, original_data, version_tag):
         for i in range(len(original_data)):
             if original_data[i]["version"] == version_tag:
+                if original_data[i]["cve_details"] != "Not Available":
+                    version_data_new["cve_details"] = original_data[i]["cve_details"]
```
[the remark this answers is not in the document]
Comment (asa1997) [reply besecure_developer_toolkit/src/CreateVersionData.py:48]: @sudhirverma When overwriting you can replace the cve_details field to Not Available. As of now we dont have the code to fetch cve details.

**Review by asa1997 — CHANGES_REQUESTED** [review]
@sudhirverma Please address the comments

**Comment by asa1997** [comment]
@sudhirverma Please resolve the conflicts

**Comment by sudhirverma** [comment]
Closing this issue as fixed in #15

**Pull request closed without merging**